From: Maximiliano Curia <maxy@debian.org>
Date: Sat, 7 Nov 2015 16:00:53 +0100
Subject: clang_FTBFS_Wreturn-type

---
 src/model/item-data.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: debian-oregano/src/model/item-data.c
===================================================================
--- debian-oregano.orig/src/model/item-data.c
+++ debian-oregano/src/model/item-data.c
@@ -453,8 +453,8 @@ void item_data_set_property (ItemData *d
 {
 	ItemDataClass *id_class;
 
-	g_return_if_fail (data != NULL);
-	g_return_if_fail (IS_ITEM_DATA (data));
+	g_return_val_if_fail (data != NULL, NULL);
+	g_return_val_if_fail (IS_ITEM_DATA (data), NULL);
 
 	id_class = ITEM_DATA_CLASS (G_OBJECT_GET_CLASS (data));
 	if (id_class->set_property) {
